Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+        contains various widgets for displaying important information and quick
+        access.
+
+    ..  _data-processor:
+    -   *Data Processor*
+
+        A data processor is a component that processes and manipulates data
+        before it is displayed in the TYPO3 frontend. Data processors are
+        implemented as PHP code and can be executed through TypoScript
+        configuration, and affect data passed on to Fluid templates. Thus, it is
+        a specific implementation to operate on data before it is passed on to
+        the presentation layer (templates).
+
+    ..  _data-provider:
+    -   *Data Provider*
+
+        A data provider is a source of data that can be used by other components
+        within the TYPO3 system. Data providers are commonly used for passing on
+        data in the backend (for example defining available icons, item keys and
+        values).
+
+    ..  _datahandler:
+    -   *DataHandler*
+
+        The DataHandler is a central component in TYPO3 responsible for
+        processing and storing data changes. It is a large PHP class that is
+        used in the backend to receive data from the FormEngine (content
+        elements and records), and is also part of an extensive API that can be
+        used by extensions to operate on records.
+
+    ..  _db-analyzer---db-compare:
+    -   *DB Analyzer / DB Compare*
+
+        DB Analyzer and DB Compare are tools in TYPO3 that analyze and compare
+        database structures to identify needed changes on the database-level for
+        both for system upgrades and extension integration. Other systems often
+        call this "database migration".
+
+    ..  _db-mounts---mount-points:
+    -   *DB Mounts / Mount Points*
+
+        Mount points allow TYPO3 editors to mount a page (and its subpages) from
+        a different area in the current backend page tree.
+
+    ..  _dbal:
+    -   *DBAL*
+
+        The Database Abstraction Layer (DBAL) is an abstraction layer in TYPO3
+        that facilitates access to various database systems.
+
+*   E
+
+    ..  _enhancer:
+    -   *Enhancer*
+
+        An enhancer is a component that adds additional functionalities or
+        improvements to existing TYPO3 features, most commonly used for
+        "Routing Enhancers" operating on speaking URL fragments.
+
+    ..  _exclude-fields:
+    -   *Exclude Fields*
+
+        Exclude fields are fields that have been configured to allow being
+        hidden in the TYPO3 backend for specific users or user groups. This is
+        done via the permission setup.
+
+    ..  _extbase:
+    -   *Extbase*
+
+        Extbase is a framework for developing extensions in the TYPO3 system
+        based on the Model-View-Controller (MVC) principle. It allows to easily
+        define models with their data fields, that can be managed by editors in
+        the backend. The models can then also be revealed in the frontend with
```

Comment:
Models can not be managed by editors. Models contains properties, not fields. Editors can only edit columns or fields, if configured in TCA, but not because it is part of the model.

Reply:
This applies to "data fields", which should better say "data records". These can be managed by editors, was what Inwanted to express